src/js/languages/fa.js

```diff
@@ -0,0 +1,90 @@
+/*!
+ * This is a `i18n` language object.
+ *
+ * Persian
+ *
+ * @author
+ *   wmateam (GitHub: @wmateam)
+ *
+ * @see core/i18n.js
+ */
+
+(function (exports) {
+    if (exports.fa === undefined) {
+        exports.fa = {
+            "mejs.plural-form": 1,
```

Comment:
According to `src/js/core/i18n.js`, Persian plural form is zero. Just change that to make sure no issues will come up when translating plurals